crates/node-resource-metrics/src/collectors/basic_node_info_collector.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::collectors::common::NAMESPACE;
+use aptos_infallible::Mutex;
+use aptos_metrics_core::const_metric::ConstMetric;
+use prometheus::{
+    core::{Collector, Desc, Describer},
+    proto::MetricFamily,
+    Opts,
+};
+use std::sync::Arc;
+use sysinfo::{RefreshKind, System, SystemExt};
+
+const BASIC_NODE_INFO_METRICS_COUNT: usize = 2;
+const RELEASE_GIT_HASH_LABEL: &str = "release_git_hash";
+const NODE_HOST_NAME_LABEL: &str = "host_name";
+
+const GIT_HASH_LABEL: &str = "git_hash";
+const HOST_NAME_LABEL: &str = "name";
+
+const UNKNOW_LABEL: &str = "unknown";
+
+pub(crate) struct BasicNodeInfoCollector {
+    system: Arc<Mutex<System>>,
+    release: Desc,
+    host_name: Desc,
+}
+
+impl BasicNodeInfoCollector {
+    fn new() -> Self {
+        let system = Arc::new(Mutex::new(System::new_with_specifics(RefreshKind::new())));
+
+        let release = Opts::new(RELEASE_GIT_HASH_LABEL, "Release git hash.")
+            .namespace(NAMESPACE)
+            .variable_label(GIT_HASH_LABEL)
+            .describe()
+            .unwrap();
+        let host_name = Opts::new(NODE_HOST_NAME_LABEL, "Host name.")
```

Comment:
nit hostname not host name: https://en.wikipedia.org/wiki/Hostname

Reply:
I think part of the issue is that sys_info uses `host_name`: https://docs.rs/sysinfo/latest/sysinfo/struct.System.html 😄

Reply:
I am hoping this is accurate enough for our use case. Will see :)